code/game/objects/items/weapons/abridged_tome.dm

```diff
@@ -0,0 +1,532 @@
+		user << "<span class='cult'>Used to scribe certain runes, and can also be used as a powerful burn weapon.</span>"
+
+/obj/item/weapon/abridged_tome/attack(mob/living/M, mob/living/user)
+	if(!is_reader(user) && !iscultist(user))
+		return ..()
+	if(iscultist(M))
+		if(M.reagents && M.reagents.has_reagent("holywater"))
+			user << "<span class='cult'>You remove the holy water from [M].</span>"
+			var/holy2unholy = M.reagents.get_reagent_amount("holywater")
+			M.reagents.del_reagent("holywater")
+			M.reagents.add_reagent("unholywater",holy2unholy)
+			add_logs(user, M, "smacked", src, " removing the holy water from them")
+		return
+	M.take_bodypart_damage(0, rand(5, 20)) //Damage done is a throwback to the damage that actual tomes used to do
+	playsound(M, 'sound/weapons/sear.ogg', 50, 1)
+	M.visible_message("<span class='danger'>[user] strikes [M] with the arcane tome!</span>", \
+					  "<span class='userdanger'>[user] strikes you with the tome, searing your flesh!</span>")
+	user.do_attack_animation(M)
+	add_logs(user, M, "smacked", src)
+
+/obj/item/weapon/abridged_tome/attack_self(mob/living/user)
+	if(!is_reader(user) && !iscultist(user))
+		if(!reader)
+			user << "<span class='notice'>You cut open a finger and press a droplet of blood onto [src]'s pages. You can read them now.</span>"
+			user.take_bodypart_damage(0.1)
+			reader = user.mind
+			return
+		user << "<span class='warning'>[src] seems full of unintelligible shapes, scribbles, and notes. They all seem particularly boring.</span>"
+		return
+	open_tome(user)
+
+/obj/item/weapon/abridged_tome/proc/open_tome(mob/user)
+	switch(alert(user,"You open the tome...",,"Scribe Rune","More Information","Cancel"))
+		if("Scribe Rune")
+			scribe_rune(user)
+		if("More Information")
+			read_tome(user)
+		if("Cancel")
+			return
+
+/obj/item/weapon/abridged_tome/proc/read_tome(mob/user)
+	var/text = "<i>There's an old introduction to aspiring cultists here, but it seems to have been scribbled over. There's some stuff written in a footnote below it...</i><br><br>"
+	text += "<font size=3><b>This version of the tome is abridged so you don't go insane from reading it. Below is a list of the runes you can write with this one.</b></font><br><br>"
+	text += "<font color='red'><b>Astral Communion</b></font> is the only rune in here that isn't watered down from its actual version. Using it will allow you to wander the spirit realm as a \
+	ghost for as long as you're alive. Bear in mind that your actual body is completely vulnerable, and will constantly take damage while you're a spirit.<br><br>"
+	text += "<font color='red'><b>Fry Circuits</b></font> sends out a moderately-sized EMP blast.<br><br>"
+	text += "<font color='red'><b>Make Talisman</b></font> functions like the normal Create Talisman rune, but the talismans it creates are weaker. To use, write the rune and put some paper \
```

Comment:
How? Or is this just a duplicated version of how they used to work? Iirc you put paper on, make a rune nearby, slap rune. If it changed I had no idea, sorry!